src/lightning/fabric/strategies/deepspeed.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         load_full_weights: bool = False,
         precision: Optional[Precision] = None,
         process_group_backend: Optional[str] = None,
+        **kwargs: Any,
```

Comment:
The other strategies have `timeout` as an explicit argument (https://github.com/Lightning-AI/pytorch-lightning/blob/be608fa355b835b9b0727df2f5476f0a1d90bc59/src/lightning/fabric/strategies/fsdp.py#L144).

I strongly suggest we do that here as well, adding `kwargs` hampers discoverability.

Let's keep the pattern exactly the same as FSDP.

Reply:
That makes sense. I've updated the PR.

Reply:
looks good thanks!